inst/include/Rcpp/sugar/matrix/lower_tri.h

```diff
@@ -1,8 +1,9 @@
-// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; tab-width: 8 -*-
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
 //
 // lower_tri.h: Rcpp R/C++ interface class library -- lower.tri
 //
-// Copyright (C) 2010 - 2011 Dirk Eddelbuettel and Romain Francois
+// Copyright (C) 2010 - 2017 Dirk Eddelbuettel and Romain Francois
```

Comment:
I think one usually separates the years, so that would be 2010 - 2016 if the next one 2017 - ...   No biggie.